static/css/restyle/restyle.less

```diff
@@ -1516,7 +1516,7 @@ h3.author .transfer-ownership {
     position: relative;
 
     &:nth-of-type(3n) .persona.hovercard:hover {
-      margin-bottom: -3px;
+      margin-bottom: -14px;
```

Comment:
I tried it on my retina Mac and my non-retina Windows VM so it seemed good. Is there a fingers-crossed emoji?